provider/datagen/Cargo.toml

```diff
@@ -40,11 +40,13 @@ eyre = "0.6"
 icu_calendar = { version = "0.5", path = "../../components/calendar", features = ["datagen"] }
 icu_casemapping = { version = "0.1", path = "../../experimental/casemapping", features = ["provider_serde", "provider_transform_internals"], optional = true }
 icu_codepointtrie = { version = "0.3.3", path = "../../utils/codepointtrie", features = ["serialize"] }
+icu_collator = { version = "0.5", path = "../../experimental/collator" }
 icu_datetime = { version = "0.5", path = "../../components/datetime", features = ["datagen"] }
 icu_decimal = { version = "0.5", path = "../../components/decimal", features = ["datagen"] }
 icu_list = { version = "0.5", path = "../../components/list", features = ["datagen"]}
 icu_locale_canonicalizer = { version = "0.5", path = "../../components/locale_canonicalizer", features = ["datagen"] }
 icu_locid = { version = "0.5", path = "../../components/locid", features = ["std"]}
+icu_normalizer = { version = "0.5", path = "../../experimental/normalizer" }
```

Comment:
Experimental functionality should be protected by the `experimental` feature. I'm fixing this for the two existing experimental crates in #1796, please take a look at that.

Reply:
Thanks. It looks like it makes sense to let that PR land before taking action here.

Reply:
#1796 has been merged

Reply:
Done.